pkg/security/module/self_tester.go

```diff
@@ -124,10 +124,10 @@ func (t *SelfTester) AddSelfTestRulesToRuleSets(ruleSet, approverRuleSet *rules.
 	}
 }
 
-// RunSelfTest runs the self test and send a report
-func (t *SelfTester) RunSelfTest(m *Module) error {
+// RunSelfTest runs the self test and return the result
+func (t *SelfTester) RunSelfTest() (error, []string, []string) {
```

Comment:
This is great, let's just move the error to the end to be more idiomatic

Reply:
Done :)